best_practices/forms.rst

```diff
@@ -91,6 +93,7 @@ directly in your form class, this would effectively limit the scope of that form
 
 .. code-block:: php
 
+    // src/AppBundle/Form/Type/PostType.php
```

Comment:
We should add the `namespace` declaration and after that add a placeholder comment or add a `use` statement for the `AbstractType` class.